src/test/java/edu/oakland/test/middleware02/RequestRfidTest.java

```diff
@@ -0,0 +1,23 @@
+package edu.oakland.test.middleware02;
```

Comment:
For your sanity in terms of getting this pull request merged, and my sanity in terms of keeping track of design vs. implementation, can we move this an the other Java files to another PR? You should be able to cut them to another folder (outside the repo), commit the change, the set up a new branch and paste them in there to remove them from this PR.